### DISCUSSION TITLE
1.20.0 beta

### DISCUSSION THREAD

**Discussion by jamie-mh** [stratumauth/app#764] [General]
**Beta release, may contain bugs. This can be installed alongside a regular release.**

- **New feature**: Icon packs
- **New feature**: Black theme option
- **New feature**: Edit item details
- **New feature**: Quick add to categories
- **New feature**: Hide usernames
- **General**: 3 new icons
- **General**: Translation updates

Download icon packs:
[https://github.com/jamie-mh/AuthenticatorProIcons/releases](https://github.com/jamie-mh/AuthenticatorProIcons/releases)

<hr /><em>This discussion was created from the release <a href='https://github.com/jamie-mh/AuthenticatorPro/releases/tag/1.20.0-beta'>1.20.0 beta</a>.</em>

**Comment by j-perezr** [comment]
Yesss! Thank you for implementing the "edit details" feature
